code/modules/crafting/recipes.dm

```diff
@@ -71,12 +71,11 @@
 				/obj/item/clothing/suit/armor/vest = 1,
 				/obj/item/robot_parts/l_leg = 1,
 				/obj/item/robot_parts/r_leg = 1,
-				/obj/item/stack/sheet/metal = 5,
-				/obj/item/stack/cable_coil = 5,
+				/obj/item/stack/sheet/metal = 1,
+				/obj/item/stack/cable_coil = 1,
 				/obj/item/weapon/gun/energy/gun/advtaser = 1,
 				/obj/item/weapon/stock_parts/cell = 1,
-				/obj/item/device/assembly/prox_sensor = 1,
-				/obj/item/robot_parts/r_arm = 1)
+				/obj/item/device/assembly/prox_sensor = 1)
```

Comment:
This looks like it adjusts the recipe to consume a single "item stack" of metal and cable coils each, as well as remove the `r_arm` from the list of components. Was there something to the arm that was making this robot uncraftable?